sdk/core/azure-core-tracing-opencensus/src/samples/AsyncQueueClientEnqueueMessages.md

```diff
@@ -7,29 +7,31 @@ messages on queue client with tracing instrumentation.
 Sample uses **[opencensus-impl][opencensus_impl]** as implementation package and **[Zipkin Exporter][zipkin_exporter]** as exporter.
 
 ### Adding dependencies to your project:
+
 [//]: # ({x-version-update-start;com.azure:azure-storage-queue;current})
 ```xml
 <!-- Add Storage Queue dependency -->
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-queue</artifactId>
-  <version>12.1.0</version> <!-- {x-version-update;com.azure:azure-storage-queue;current} -->
+  <version>12.1.0</version>
 </dependency>
 ```
 [//]: # ({x-version-update-end})
+
```

Comment:
You don't need the extra line in between the two version tags ([//]:) only before/after the entire block in these cases. That's a general comment for the .md files.